packages/tables/src/Actions/Action.php

```diff
@@ -69,18 +78,38 @@ public function getLivewireClickHandler(): ?string
             return null;
         }
 
-        return $this->generateJavaScriptClickHandler('mountTableAction');
+        $arguments = $this->getArguments();
+
+        return $this->generateJavaScriptClickHandler('mountTableAction', [
+            ...$arguments ? [$arguments] : [],
+        ]);
     }
 
-    protected function generateJavaScriptClickHandler(string $method): string
+    /**
+     * @param  array<array-key, array<mixed, mixed>>  $parameters
+     */
+    protected function generateJavaScriptClickHandler(string $method, array $parameters = []): string
```

Comment:
Why do we need to pass the params in, can we not fetch them inside the method? Like we do for the main Action class?

Reply:
Yes we can, but in theory you can also pass in a different method then `mountTableAction`, and we wouldn't be sure if such other method would also accept `$arguments`. 

Though to be fair that also applies to the `$record` parameter, if you pass in a different method into `generateJavaScriptClickHandler()` then Filament also cannot be sure that the 2nd parameter is `$record`.

Reply:
Another option would be to rename `generateJavaScriptClickHandler()` into something like `getJavascriptMountTableActionClickHandler()` to make it more explicit that this method is *only* intended to be used for `mountTableAction()`.

Reply:
The method for the Livewire and JavaScript click handlers will always have the same signature, I thought the difference was that one was prefixed with `$wire.`...